tests/load/README.md

```diff
@@ -219,6 +208,128 @@ Execute the `setup_k8s.sh` file from the root directory and select the **delete*
 ./tests/load/setup_k8s.sh
 ```
 
+## Calibration
+
+Following the addition of new features, such as a Locust Task or Locust User, or environmental 
+changes, such as node size it may be necessary to re-establish the recommended parameters of a 
```

Comment:
```suggestion
changes, such as node size or the upgrade of a major dependency like the python version image, it may be necessary to re-establish the recommended parameters of a 
```

thoughts?